infra/k8s/cluster.yaml.example_with_monitoring

```diff
@@ -0,0 +1,108 @@
+apiVersion: kops.k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: null
+  name: my-first-cluster-kops.k8s.local
+spec:
+  addons:
+  - manifest: metrics-server
+  - manifest: prometheus-operator
+  - manifest: kubernetes-dashboard
+  api:
```

Comment:
on my end, `kops` generates:

```
  api:
    loadBalancer:
      type: Public
```

Why are we changing it to `dns: {}` and what does that mean?

Reply:
Looks like you reverted this, so I am resolving this conversation.